pkg/gci/internal/testdata/grouped-multiple-custom.cfg.yaml

```diff
@@ -0,0 +1,4 @@
+sections:
+  - Standard
+  - Default
+  - Prefix(github.com/daixiang0;daixiang0)
```

Comment:
Not comma?

Reply:
The biggest issue when you fix something is that you are too confident and do not review everything again :sweat_smile: 

I will fix everything